src/components/challengeWidget/challengeWidget.css

```diff
@@ -0,0 +1,32 @@
+.challengeWidget {
+  width: 700px;
+  height: 250px;
+  border: 15px solid burlywood;
+  background-color: white;
+}
+
+h1 {
+  color: black;
+  font-family: sans-serif;
+}
+p {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.first_button {
+  float: right;
+  padding: 5px;
+}
+.paragraph {
+  font-family: serif;
+}
+.cup {
+  border-radius: 50%;
+  display: inline-block;
+}
+.curcles {
+  display: inline;
+  display: flex;
+  padding: 2px;
+  justify-content: space-between;
+  border-radius: 50%;
```

Comment:
I don't think this is needed on the container.

Reply:
I didn't understand what do you mean, if I deleted this code the item will not have space between, I am not sure what to do about it please give me a hint